### ISSUE TITLE
Adding a new note to a notebook directly should be present for easy accessibility 

### ISSUE THREAD

**Issue by Suswetha6** [priyasirohi09/joplin#282]
OS : LINUX
Current Behavior: 
Whenever we create a notebook,there is no direct way to add a note inside that.We would have to create a note and manually take it and place inside the folder.It would improve the UI if we could have a button beside the folder created to create a note directly inside the folder 
Screenshot: 
![image](https://github.com/user-attachments/assets/b6103f83-3975-4e2d-92a7-fd1e1a8614e6)
